src/main/java/scrolls/elder/logic/parser/AddressBookParser.java

```diff
@@ -92,6 +94,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case LogAddCommand.COMMAND_WORD:
             return new LogAddCommandParser().parse(arguments);
 
+        case UndoCommand.COMMAND_WORD:
```

Comment:
Do we want to consider alternative command words for the UndoCommand? We can possibly consider "reverse", "cancel" etc.

Reply:
That's a possible alternative as well, maybe we can discuss it and I can make the changes accordingly.